### ISSUE TITLE
Interested in collaborating

### ISSUE THREAD

**Issue by ghost** [K1ish/mmswebproxy#1]
I'm interested in collabing on this! The concept is really unique, and it sounds like maybe you'd need some help with the front end. I already see a couple of issues with this, such as legal and database-scope problems, but I'm sure there could be a workaround.

I can code in a myriad of programming languages, and I learn fast (humble brag? I don't intend it that way). With that, I would love to help!

You might remember my school email... check your "Sent" inbox to find it if you don't remember my student ID. if you're interested, shoot me an email there! I'll reply with contact details I use more often.

Thanks man!

**Comment by K1ish** [comment]
Yeah sure! I'm doing Python for this one with Google voice for messaging.
I'm also going to determine if gvoice or email to mms proxies are faster.

**Comment by ghost** [comment]
Sorry for such a late response! Some summer stuff popped up.

I'm actually really bad at backend coding, but I enjoy UI/application design a lot more - I could start making some mockups in the meantime. We could make a private web browser app out of this (enoooormous project though, even if we base it off Chromium), or we could make it a Chrome/Firefox extension on those separate apps with a "route data" button. What are your ideas?
